src/index.html

```diff
@@ -2,11 +2,379 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport" content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <nav class="nav">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo_logo"
+            class="nav__moyo-logo"
+          >
+        </a>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link space">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <h1 class="visually_hidden">
+        Catalog
+      </h1>
+
+      <section class="conteiner">
+        <div class="box" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="box__img"
+          >
+          <div class="box__text">
+            <h2 class="box__text_title">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </h2>
+            <p class="box__text_code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="box__rate">
+            <div class="box__rate_stars stars">
+              <div class="stars_conteiner"></div>
+              <div class="stars_conteiner"></div>
+              <div class="stars_conteiner"></div>
+              <div class="stars_conteiner"></div>
+              <div class="stars_conteiner"></div>
+            </div>
+            <div class="box__rate_response">
```

Comment:
use - for long names, not _

Reply:
Why i remember that Misha say it doesn't matter?